### ISSUE TITLE
Topic tooltip is broken

### ISSUE THREAD

**Issue by akempgen** [colloquy/colloquy#543]
*Issue migrated from trac ticket [#259](http://colloquy.info/project/ticket/259)*

**type:** Defect | **component:** Colloquy (Mac) | **severity:** Minor | **priority:** Low | **version:** 2.0 (Mac) | **status:** closed | **resolution:** Fixed | **keywords:** topic tooltip
___
#### 2005-07-21 18:41:19: David created the issue

I used to hover over the topic to see a tooltip indicating who set the topic. Since you added the ellipsis button to the topic bar, the tooltip won't show up anymore.
I'm on 10.3.9, using the latest nightly build (same issue in the latest stable release, though). Keep up the good work :)
___
#### 2005-07-21 19:01:39: @kballard commented

So basically, a good solution would be to add the setter information to the ellipsis button dropdown?

___
#### 2005-07-22 21:07:26: David changed severity from *normal* to *minor*

___
#### 2005-07-22 21:07:26: David commented

Yes, that would be a good solution. (And perhaps a timestamp?)

___
#### 2005-10-12 05:27:31: @xeenon changed status from *new* to *closed*

___
#### 2005-10-12 05:27:31: @xeenon changed resolution from ** to *fixed*

___
#### 2005-10-12 05:27:31: @xeenon commented

Fixed in 12a0da9f751bda05f64fd2ee27c78661fdec579d
